MekHQ/src/mekhq/gui/dialog/CampaignOptionsDialog.java

```diff
@@ -3467,65 +3469,72 @@ public void actionPerformed(java.awt.event.ActionEvent evt) {
         gridBagConstraints.gridy = 5;
         panSubAtBAdmin.add(chkCustomRetirementMods, gridBagConstraints);
 
+        chkFoundersNeverRetire.setText(resourceMap.getString("chkFoundersNeverRetire.text"));
+        chkFoundersNeverRetire.setToolTipText(resourceMap.getString("chkFoundersNeverRetire.toolTipText"));
+        chkFoundersNeverRetire.setSelected(options.getFoundersNeverRetire());
+        gridBagConstraints.gridx = 0;
+        gridBagConstraints.gridy = 6;
```

Comment:
If you go back to where the gridbagConstraints.gridy is first used for this column and declare an int that starts at 0, then, instead of having to update every gridy after any item you added, you can just use "gridBagConstraints.gridy = intvalue++".

Reply:
Now that I think about it, you could just set gridBagConstraints.gridy to 0 at the beginning of the column, then call ++ on it for each subsequent row.

Reply:
Yeah I like this, would save us the trouble each time this gets changed.